### PR TITLE
rectify options sheets for groups/channels/dms/multi-dms

### PR DESCRIPTION
**Pull request by patosullivan**
fixes TLON-2735

### CODE REVIEW

File: packages/ui/src/components/ChatOptionsSheet.tsx [hunk 1]
```diff
@@ -584,6 +621,7 @@ function ChatOptionsSheetContent({
   subtitle: string;
   icon?: ReactElement;
 }) {
+  console.log('ChatOptionsSheetContent', actionGroups, title, subtitle, icon);
```
Comment (dnbrwstr) [inline packages/ui/src/components/ChatOptionsSheet.tsx:624]: Stray log

**Review by dnbrwstr — APPROVED** [review]
Lgtm!

**Comment by jamesacklin** [comment]
Looks good. A few things:

image

A) "Pin" should exist in group -> channel options, but should exist for DMs, multi-DMs, and root-level groups (see first two screens).

B) I'd love it if we could move "Edit group info" in the multi-DM option sheet to its own single-item section to stay consistent with the others.

**Comment by jamesacklin** [comment]
Pin should stay for channels; the fact that they don't show up in the chat list is a separate bug.

Confirmed "Edit group info" is its own island for multi-DMs:

image

Confirmed "Invite people" shows up for my secret group:

image

**Review by jamesacklin — APPROVED** [review]

**Pull request merged**